File: server-scala-play-server/app/model/PaymentTypesResponse.scala
package model

import play.api.libs.json._

/**
  * Represents the Swagger definition for PaymentTypesResponse.
  */
@javax.annotation.Generated(value = Array("org.openapitools.codegen.languages.ScalaPlayFrameworkServerCodegen"), date = "2026-02-09T20:57:39.376804970Z[Etc/UTC]", comments = "Generator version: 7.20.0-SNAPSHOT")
case class PaymentTypesResponse(
  valid: Option[Boolean],
  message: Option[String],
  version: Option[Double],
  serializeNulls: Option[Boolean],
  startTimeLog: Option[Long],
  errorCode: Option[String],
  request: Option[List[NameStringValueResponse]],
  creditCardOnFile: Option[Boolean],
  creditCardNumber: Option[String],
  creditCardExpirationMonth: Option[String],
  creditCardExpirationYear: Option[String],
  pointsOnFile: Option[Boolean],
  pointsAmount: Option[Double],
  account: Option[AccountShortResponse],
  firstName: Option[String],
  lastName: Option[String],
  address: Option[String],
  city: Option[String],
  state: Option[String],
  postalCode: Option[String],
  phone: Option[String],
  provider: Option[String],
  paymentMethodId: Option[Long],
  validated: Option[Boolean],
  accountNumber: Option[String],
  bankName: Option[String],
  routingNumber: Option[String],
  isDefaultPaymentMethod: Option[Boolean],
  accountName: Option[String],
  paymentMethodNickname: Option[String],
  taxId: Option[String],
  returning: Option[String]
)

object PaymentTypesResponse {
  implicit lazy val paymentTypesResponseJsonFormat: Format[PaymentTypesResponse] = Json.format[PaymentTypesResponse]
}

File: server-aspnet-fastendpoints/src/Org.OpenAPITools/Models/PaymentTypesResponse.cs
namespace Org.OpenAPITools.Models;


/// <summary>
/// 
/// </summary>
public class PaymentTypesResponse 
{
    public bool Valid { get; set; }
    public string Message { get; set; }
    public double VarVersion { get; set; }
    public bool SerializeNulls { get; set; }
    public long StartTimeLog { get; set; }
    public string ErrorCode { get; set; }
    public List<NameStringValueResponse> Request { get; set; }
    public bool CreditCardOnFile { get; set; }
    public string CreditCardNumber { get; set; }
    public string CreditCardExpirationMonth { get; set; }
    public string CreditCardExpirationYear { get; set; }
    public bool PointsOnFile { get; set; }
    public double PointsAmount { get; set; }
    public AccountShortResponse Account { get; set; }
    public string FirstName { get; set; }
    public string LastName { get; set; }
    public string Address { get; set; }
    public string City { get; set; }
    public string State { get; set; }
    public string PostalCode { get; set; }
    public string Phone { get; set; }
    public string Provider { get; set; }
    public long PaymentMethodId { get; set; }
    public bool Validated { get; set; }
    public string AccountNumber { get; set; }
    public string BankName { get; set; }
    public string RoutingNumber { get; set; }
    public bool IsDefaultPaymentMethod { get; set; }
    public string AccountName { get; set; }
    public string PaymentMethodNickname { get; set; }
    public string TaxId { get; set; }
    public string Returning { get; set; }
}


